integration-tests/jms/src/main/resources/application.properties

```diff
@@ -1,2 +1,2 @@
-quarkus.artemis.xa.enabled=true
-quarkus.artemis.url=tcp://localhost:61616
+quarkus.artemis.devservices.enabled=false
```

Comment:
This should not be necessary, right?

Reply:
Well, it is compatible with using `ArtemisTestResouce` which starts/stops `EmbeddedActiveMQ`. I think we need to mention in the document that if users still want to use `ArtemisTestResource`, it needs to set `quarkus.artemis.devservices.enabled=false` or `quarkus.artemis.url` explictly.

IIRC we change to use `EmbeddedActiveMQ` dues to [vromero/activemq-artemis-docker](https://github.com/vromero/activemq-artemis-docker) does not maintain anymore.

I'm not sure that [artemiscloud/activemq-artemis-broker](https://quay.io/repository/artemiscloud/activemq-artemis-broker) could be supported by Red Hat offically? @gastaldi @gsmet 

I think we might add `@Deprecated` on `ArtemisTestResource` and will drop it in the future 2.x?

Reply:
[vromero was never active](https://github.com/vromero/activemq-artemis-helm/pull/25), we have been using our own images since then. Good point about `ArtemisTestResource`, it is actually used by some other projects. I have no clear view about the usage. We can keep both, it is a small class.